src/ImageSharp/Formats/Bmp/BmpFileHeader.cs

```diff
@@ -44,12 +48,25 @@ public BmpFileHeader(short type, int fileSize, int reserved, int offset)
         /// Gets any reserved data; actual value depends on the application
         /// that creates the image.
         /// </summary>
-        public int Reserved { get;  }
+        public int Reserved { get; }
 
         /// <summary>
         /// Gets the offset, i.e. starting address, of the byte where
         /// the bitmap data can be found.
         /// </summary>
         public int Offset { get; }
+
+        public static BmpFileHeader Parse(Span<byte> data)
+        {
+            return MemoryMarshal.Cast<byte, BmpFileHeader>(data)[0];
+        }
+
+        public unsafe void WriteTo(Span<byte> buffer)
+        {
```

Comment:
How about this without pinning? 
```C#
ref BmpFileHeader dest = ref Unsafe.As<byte, BmpFileHeader>(ref MemoryMarshal.GetReference(buffer));
dest = this;
```
Doesn't really matter here, but it's most likely faster :)

Reply:
Much nicer!